x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_settings_collector.js

```diff
@@ -56,6 +55,7 @@ export async function checkForEmailValue(
 
 export function getSettingsCollector(server) {
   const config = server.config();
+  const { Collector } = server.usage;
```

Comment:
This is creative. I wonder if this approach will work with the future plugin system / TypeScript?

Reply:
Earlier ago, I talked with @epixa about these changes and he let me know that if we're registering server methods as per the conventions from the Hapi ecosystem, we should be OK for at least the foreseeable future.

I mentioned Court on this so he can correct me if I misunderstood.

Reply:
Yep, this should be fine.